resources/js/src/app/helper/StickyElement.js

```diff
@@ -23,7 +23,16 @@ export class StickyElement
         this.isMinWidth = true;
         this.checkMinWidth();
 
-        this.resizeListener = this.checkMinWidth.bind(this);
+        this.resizeListener = () =>
+        {
+            this.checkMinWidth();
+
+            if (this.enabled && this.position.isSticky)
+            {
+                this.checkElement();
+                this.updateStyles();
+            }
```

Comment:
Warum wird dieser Block benötigt? Innerhalb von `enable()` wird in Zeile 63 auf alle `STICKY_EVENTS` ein EventListener gesetzt, der letztendlich dieselbe Logik beinhaltet. In diesen Events ist auch `resize` enthalten...

Reply:
done